templates/plugin/container_gateway.yml.erb

```diff
@@ -2,7 +2,15 @@
 # Container Gateway for Katello
 :enabled: <%= @module_enabled %>
 :pulp_endpoint: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::pulp_endpoint") %>
+:database_backend: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::database_backend") %>
 :sqlite_db_path: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_db_path") %>
 <% if scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") -%>
 :sqlite_timeout: <%= scope.lookupvar("foreman_proxy::plugin::container_gateway::sqlite_timeout") %>
 <% end -%>
+:postgresql_connection_string: postgres://<%=
```

Comment:
I see it was already merged and released, but I still would insist on a simple single parameter in the settings file while getting rid of the explicit `database_backend`. https://github.com/Katello/smart_proxy_container_gateway/pull/36 does that.

Reply:
That's fine, we can get that in first.